geometric_kernels/lab_extras/extras.py

```diff
@@ -52,3 +52,28 @@ def sinh(x: B.Numeric) -> B.Numeric:
     .. math:: cosh(x) = \frac{\exp(x) - \exp(-x)}{2} ,
     """
     return 0.5 * (B.exp(x) - B.exp(-x))
+
+
+@dispatch
+@abstract()
+def degree(a):
+    """
+    Given a vector a, return a diagonal matrix with a as main diagonal.
```

Comment:
I don't think the description matches with the impementation.
Also, can it be done with pure lab, something along the lines of:
```
degrees = B.sum(a, axis=0)
return B.diag(degrees)
```
That way it'd be easier to maintain?